packages/libs/eda/src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -1102,6 +1102,11 @@ function LineplotViz(props: VisualizationProps<Options>) {
     !showDependentAxisBanner &&
     data0?.binWidthSlider == null; // for ordinal string x-variables
 
+  // always enable useBinning for timeline Viz
+  const alwaysEnableUseBinning = useMemo(() => {
```

Comment:
Not sure a `useMemo` is needed here.  No heavy calculations!

Reply:
@bobular Thank you for your reviews 👍 I was debating whether it uses useMemo or not but can remove it :)